docs/source/conf.py

```diff
@@ -209,6 +209,7 @@
      'examples_dirs': '../../examples',
      # path where to save gallery generated examples
      'gallery_dirs': 'auto_examples',
+     'ignore_pattern': 'plot_baby_afq.py',
```

Comment:
I wonder why this is not in the `exclude_patterns` on line 82 of this file (and what the difference is between these), but maybe that's not a specific issue with this PR

Reply:
I don't recall exactly why I made this change, it was part of the original commit, so it could be OBE